include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -873,6 +873,9 @@ struct SessionOptionsImpl : ConstSessionOptionsImpl<T> {
 
   SessionOptionsImpl& AddInitializer(const char* name, const OrtValue* ort_val);                                             ///< Wraps OrtApi::AddInitializer
   SessionOptionsImpl& AddExternalInitializers(const std::vector<std::string>& names, const std::vector<Value>& ort_values);  ///< Wraps OrtApi::AddExternalInitializers
+  SessionOptionsImpl& AddExternalInitializerFiles(const std::vector<std::basic_string<ORTCHAR_T>>& file_names,
```

Comment:
It seems that in other C++ API, file names are usually used as type of `const ORTCHAR_T*`. (eg. `SetOptimizedModelFilePath()` )

Is there a preferred way for representing file path in function parameters?

Reply:
how do you maintain a vector<ORTCHAR_T*>?

Reply:
Maybe the owner of C++ API can answer this question. I don't have a preference, but I think it should be consistent.

Reply:
depending on consistent to which one. Like the AddExternalInitializers(const std::vector<std::string>& names...

Reply:
I believe internally we should use PathString type that is alrady defined in common layer.

Reply:
updated.